api/src/main/resources/application.properties

```diff
@@ -4,4 +4,4 @@ spring.datasource.username=postgres
 spring.datasource.password=memery
 
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto = update
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto = validate
```

Comment:
Changed from update to validate so that flyway can migrate instead of hibernate automatically doing it. The `validate` will throw an error if the JPA models and database tables don't match